### PR TITLE
fix: sensor to use scheduled_time & all the window params

### PR DESCRIPTION
**Pull request by sravankorumilli**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,24 +7,17 @@ LAST_TAG := "$(shell git rev-list --tags --max-count=1)"
...
 	@echo " > building optimus version ${OPMS_VERSION}"
 	@go build -ldflags "-X ${NAME}/config.Version=${OPMS_VERSION} -X ${NAME}/config.BuildCommit=${LAST_COMMIT}" -o optimus .
 	@echo " - build complete"
 	
 test: smoke-test unit-test vet ## run tests
 
-generate: pack-files
-	@echo " > notice: skipped proto generation, use 'generate-proto' make command"
-	
-pack-files:
```
Comment (sbchaos) [inline Makefile:24]: We need to also update the github actions to remove the invocation of pack-files.
https://github.com/odpf/optimus/blob/main/.github/workflows/lint.yml#L21

File: ext/scheduler/airflow2/resources/__lib.py [hunk 2]
```diff
@@ -388,12 +392,12 @@ def _parse_datetime(self, timestamp) -> datetime:
         except ValueError:
             return datetime.strptime(timestamp, self.TIMESTAMP_MS_FORMAT)
 
-    def generate_window(self, execution_date, window_size):
+    def generate_window(self, schedule_time):
         format_rfc3339 = "%Y-%m-%dT%H:%M:%SZ"
-        execution_date_str = execution_date.strftime(format_rfc3339)
+        schedule_time_str = schedule_time.strftime(format_rfc3339)
         # ignore offset & truncate to
```
Comment (sbchaos) [inline ext/scheduler/airflow2/resources/__lib.py:398]: Commment is out of sync with code

File: Makefile (hunk 1 above)
Comment (sravankorumilli) [reply Makefile:22]: done

File: ext/scheduler/airflow2/resources/__lib.py (hunk 2 above)
Comment (sravankorumilli) [reply ext/scheduler/airflow2/resources/__lib.py]: done

**Pull request merged**